.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
```

Comment:
Maybe one comment regarding the title? The convention we follow (https://www.conventionalcommits.org/en/v1.0.0/), the type of titles we accept?

Reply:
yup, replied here
https://github.com/OffchainLabs/arb-token-bridge/pull/512#discussion_r1035758816